persistqueue/sqlackqueue.py

```diff
@@ -113,16 +112,14 @@ def _mark_ack_status(self, key, status):
 
     @sqlbase.with_conditional_transaction
     def clear_acked_data(self):
-        sql = """DELETE FROM {table_name}
-            WHERE {key_column} IN (
-                SELECT _id FROM {table_name} WHERE status = ?
-                ORDER BY {key_column} DESC
-                LIMIT 1000 OFFSET {max_acked_length}
-            )""".format(table_name=self._table_name,
-                        key_column=self._key_column,
-                        max_acked_length=self._MAX_ACKED_LENGTH)
+        sql = """DELETE FROM {table_name} WHERE status = ?""".format(table_name=self._table_name)
```

Comment:
I'm a little hesitant to use `format` to make sql query (bobby tables), but I suppose _table_name is safe and was already being used.